airbyte-integrations/connectors/source-instagram/unit_tests/test_streams.py

```diff
@@ -102,74 +164,38 @@ def test_user_insights_state(api, user_insights, values, slice_dates, expected):
-        "oauth_error",
-        "rate_limit_error",
-        "too_many_request_error",
-        "reduce_amount_of_data_error",
-        "unknown_error",
-        "viewers_insights_error",
-        "4028_issue_error",
-        "transient_error",
-        "user_media_creation_time_error",
+        pytest.param({"json": {"error": {"type": "OAuthException", "code": 1}}}, id="test_oauth_error"),
+        pytest.param({"json": {"error": {"code": 4}}}, id="test_rate_limit_error"),
+        pytest.param({"json": {}, "status_code": 429}, id="test_too_many_request_error"),
+        pytest.param(
+            {
+                "json": {"error": {"code": 1, "message": "Please reduce the amount of data you're asking for, then retry your request"}},
+                "status_code": 500,
+            },
+            id="test_reduce_amount_of_data_error",
+        ),
+        pytest.param({"json": {"error": {"code": 1, "message": "An unknown error occurred"}}, "status_code": 500}, id="test_unknown_error"),
+        pytest.param(
+            {
+                "json": {
+                    "error": {"type": "OAuthException", "message": "(#10) Not enough viewers for the media to show insights", "code": 10}
+                }
+            },
+            id="test_viewers_insights_error",
+        ),
+        pytest.param({"json": {"error": {"code": 100, "error_subcode": 33}}, "status_code": 400}, id="test_4028_issue_error"),
+        pytest.param({"json": {"error": {"is_transient": True}}}, id="test_transient_error"),
+        pytest.param({"json": {"error": {"code": 4, "error_subcode": 2108006}}}, id="test_user_media_creation_time_error"),
     ],
 )
-def test_common_error_retry(config, error_response, requests_mock, api, account_id, user_insight_data):
-    """Error once, check that we retry and not fail"""
-    # response = {"business_account_id": "test_id", "page_id": "act_unknown_account"}
-    responses = [
-        error_response,
-        {
-            "json": user_insight_data,
-            "status_code": 200,
-        },
-    ]
-    test_id = "test_id"
-    requests_mock.register_uri("GET", FacebookSession.GRAPH + f"/{FB_API_VERSION}/{test_id}/insights", responses)
-
-    stream = UserInsights(api=api, start_date=config["start_date"])
+@freezegun.freeze_time("2025-07-01T23:59:59Z")
+def test_common_error_retry(components_module, error_response, requests_mock, account_id):
+    config = {"start_date": "2025-07-01T00:00:00Z", "access_token": "unknown_token"}
+    mock_accounts_request(requests_mock=requests_mock, some_config=config)
+    mock_user_insights_day_request(requests_mock, {"end_time": "2025-07-01 07:00:00+0000", "value": 4}, error_response)
+    mock_user_insights_requests(requests_mock, {"end_time": "2025-07-01 07:00:00+0000", "value": 4})
+
+    stream = find_stream(stream_name="user_insights", config=config)
     records = read_full_refresh(stream)
 
     assert records
-
-
-def test_exit_gracefully(api, config, requests_mock, caplog):
```

Comment:
this test failed migrating over and also doesn't really match the exact behavior of concurrent CDK streams. it also doesn't test much of interest.